services/src/main/java/org/exoplatform/webconferencing/WebConferencingService.java

```diff
@@ -3861,7 +3870,9 @@ protected String metricMessage(String userId,
                                  String operation,
                                  String status,
                                  Long duration,
-                                 String error) {
+                                 String error,
+                                 Double file_size,
+                                 int upload_status) {
```

Comment:
As status is already present, I'm not sure we need a specific upload status.
In fact, if operation=CALL_RECORDED and status=ko, it is sufficient to know that upload failed. 
WDYT ?

Reply:
you are alright about status_not_ok because I think that status refers to the status of the call but we still need to return the status of the upload because it can happen during the upload and not inside the call for that I add this status.SO the status ok of the call does not mean that the upload is done successfully.

Reply:
the call-recorded event is only for upload, am I right ?

Reply:
Yep, but it does not exist now when uploading now that's why I implement it to continue in the other task add the metrics needed in module analytics